owncloudData/src/main/java/com/owncloud/android/data/files/repository/OCFileRepository.kt

```diff
@@ -458,11 +460,11 @@ class OCFileRepository(
         if (!anyConflictInThisFolder) {
             remoteFolder.etagInConflict = null
         }
-
-        return localFileDataSource.saveFilesInFolderAndReturnThem(
+        localFileDataSource.saveFilesInFolder(
```

Comment:
There's no need to use another method and repeat code in every layer underneath (datasource, DAO...). Just use the same method but don't use what it returns

Reply:
Done